src/core/dom/style.js

```diff
@@ -27,6 +27,14 @@ export function camelCaseToTitleCase(camelCase) {
   return camelCase.charAt(0).toUpperCase() + camelCase.slice(1);
 }
 
+/**
+ * @param {string} camelCase camel cased string
+ * @return {string} hyphen-cased string
+ */
+export function camelCaseToHyphenCase(camelCase) {
+  return camelCase.replace(/[A-Z]/g, (match) => '-' + match.toLowerCase());
```

Comment:
`webkitFoo` can be lowercase, but still needs a leading dash.

Reply:
This seems tricky

Reply:
Got it